docs/mesos/README.md

```diff
@@ -45,10 +45,13 @@ The [Calico-Mesos RPM Installation Guide](RpmInstallCalicoMesos.md) serves as th
 ### b.) Manual
 For an in-depth walkthrough of the full installation procedure performed by the RPMs, see the [Calico-Mesos Manual Install Guide](ManualInstallCalicoMesos.md).
 
-### c.) Dockerized Deployment
+### c.) Dockerized Deployment (Automated)
```

Comment:
I Think that now that we've transitioned to deploying calico-mesos "officially" via these docker images, this should really be the first method we tell people. However, I think this whole section needs a revise to more clearly outline the pros and cons of each installation method so folks have an easier means of selecting the guide thats right for them. Basically, I don't want the one thing people to use be the RPMs since its really not the best way to go, but this readme makes it seem like it is. 

However, lets make that a follow up task, since its a different scope than this PR.